### PR TITLE
chore: Updated readme to provide information on generating IAM access tokens

### PR DESCRIPTION
**Pull request by w0o**
Updated the readme to reflect the current state of IAM based access and added an example of generating IAM-based access tokens, the issue encountered in #651 .

I still think Authorization should provide this experience out of the box and in alignment with the other SDKs usage pattern without users having to use two different classes (eg. IAMTokenManager and Authorization), but now the correct method is at least documented.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,12 +137,20 @@ If you'd prefer to set authentication values manually in your code, the SDK supp
...
+IBM Cloud has migrated to token-based Identity and Access Management (IAM) authentication. IAM authentication uses a service API key to get an access token that is passed with the call. Access tokens are valid for approximately one hour and must be regenerated.
 
 You supply either an IAM service **API key** or an **access token**:
 
 - Use the API key to have the SDK manage the lifecycle of the access token. The SDK requests an access token, ensures that the access token is valid, and refreshes it if necessary.
 - Use the access token if you want to manage the lifecycle yourself. For details, see [Authenticating with IAM tokens](https://console.bluemix.net/docs/services/watson/getting-started-iam.html).
+- For untrusted environments use a server-side token generator to generate access tokens for your environment using your IAM API key. The generated access tokens will be valid for one hour and can be refreshed.
+
+### Generating access tokens using IAM API key
```
Comment (ehdsouza) [inline README.md]: This information can be added under `#### Supplying the access token`.

File: README.md [hunk 2]
```diff
@@ -137,12 +137,20 @@ If you'd prefer to set authentication values manually in your code, the SDK supp
...
 
-IBM Cloud is migrating to token-based Identity and Access Management (IAM) authentication. IAM authentication uses a service API key to get an access token that is passed with the call. Access tokens are valid for approximately one hour and must be regenerated.
+IBM Cloud has migrated to token-based Identity and Access Management (IAM) authentication. IAM authentication uses a service API key to get an access token that is passed with the call. Access tokens are valid for approximately one hour and must be regenerated.
 
 You supply either an IAM service **API key** or an **access token**:
 
 - Use the API key to have the SDK manage the lifecycle of the access token. The SDK requests an access token, ensures that the access token is valid, and refreshes it if necessary.
 - Use the access token if you want to manage the lifecycle yourself. For details, see [Authenticating with IAM tokens](https://console.bluemix.net/docs/services/watson/getting-started-iam.html).
+- For untrusted environments use a server-side token generator to generate access tokens for your environment using your IAM API key. The generated access tokens will be valid for one hour and can be refreshed.
```
Comment (ehdsouza) [inline README.md]: Maybe add the following description?
```
If you want to generate access tokens for your environment using your IAM API key. The generated access tokens will be valid for one hour and can be refreshed.
```

Comment (w0o) [reply README.md]: Perhaps the sentence should be more like the other suggestions for usage, so how about:

> Use a server-side to generate access tokens using your IAM API key for untrusted environments like client-side scripts. The generated access tokens will be valid for one hour and can be refreshed.


What do think about this?

Comment (germanattanasio) [reply README.md]: That sounds good

File: README.md [hunk 3]
```diff
@@ -137,12 +137,20 @@ If you'd prefer to set authentication values manually in your code, the SDK supp
...
 You supply either an IAM service **API key** or an **access token**:
 
 - Use the API key to have the SDK manage the lifecycle of the access token. The SDK requests an access token, ensures that the access token is valid, and refreshes it if necessary.
 - Use the access token if you want to manage the lifecycle yourself. For details, see [Authenticating with IAM tokens](https://console.bluemix.net/docs/services/watson/getting-started-iam.html).
+- Use a server-side to generate access tokens using your IAM API key for untrusted environments like client-side scripts. The generated access tokens will be valid for one hour and can be refreshed.
+
+### Generating access tokens using IAM API key
+```python
+# In your API endpoint use this to generate new access tokens
+iamTokenManager = IAMTokenManager(iam_apikey='<apikey>')
```
Comment (ehdsouza) [inline README.md]: Could this be camel cased?
`iam_token_manager `

Comment (w0o) [reply README.md]: Totally right! Updated the code sample as suggested.

**Comment by ehdsouza** [comment]
Thanks for updating the readme 👍

**Pull request merged**